web/src/user_group_popover.ts

```diff
@@ -8,32 +10,35 @@ import * as hash_util from "./hash_util";
 import * as message_lists from "./message_lists";
 import {page_params} from "./page_params";
 import * as people from "./people";
+import type {User} from "./people";
 import * as popover_menus from "./popover_menus";
 import * as rows from "./rows";
 import * as ui_util from "./ui_util";
 import * as user_groups from "./user_groups";
 import * as util from "./util";
 
-let user_group_popover_instance;
+let user_group_popover_instance: PopoverInstance | undefined;
 
-export function hide() {
+type PopoverGroupMember = User & {user_circle_class: string; user_last_seen_time_status: string};
+
+export function hide(): void {
     if (is_open()) {
-        user_group_popover_instance.destroy();
+        user_group_popover_instance!.destroy();
```

Comment:
Don’t use an unchecked non-null assertion (`!`); fix the code so TypeScript understands why the value is non-null.

```diff
-    if (is_open()) {
+    if (user_group_popover_instance !== undefined) {
```